### ISSUE TITLE
Hypothesis-pytest deprecated

### ISSUE THREAD

**Issue by Bachmann1234** [DRMacIver/hypothesis#142]
Howdy! 

Sorry if i'm just being think but I am getting this warning when running hypothesis
```
HypothesisDeprecationWarning: You have an obsolete version of the hypothesis-pytest plugin installed. Please update to a more recent version.
```

Looking at pip's output I see
```
Downloading/unpacking hypothesis-pytest==0.18.0 (from -r requirements.txt (line 4))
  Downloading hypothesis-pytest-0.18.0.tar.gz
  Running setup.py (path:/--redacted--/setup.py) egg_info for package hypothesis-pytest
```

Which seems to be the most up to date version

https://pypi.python.org/pypi/hypothesis-pytest/0.18.0

Am I missing something? Anything I should look for?

**Comment by DRMacIver** [comment]
Huh. Curious. You're not missing anything, it's just that somehow the setuptools entry point that used to be in hypothesis-pytest has snuck back in. I'm not sure how. I probably did git wrong.

I've got a fix pushed to master. Just waiting for it to build and I'll ship and updated hypothesis-pytest that fixes this.

**Comment by DRMacIver** [comment]
This should now be fixed as of 0.18.1

**Comment by Bachmann1234** [comment]
Confirmed. Thanks!
